### DISCUSSION TITLE
Is there a way to have multi-language readme.md file?

### DISCUSSION THREAD

**Discussion by Gabri432** [community/community#31132] [Other Feature Feedback, Questions, & Ideas] [answered]
### For example, is it possible to support a readme.md in english language, and another one in another language?

I may have missed this option and it might be already available, but so far it seems like I couldn't find it.

Thanks for your help. Have a nice day!

**Comment by melvincwng** [comment, marked as answer]
Hey @Gabri432, I believe it's possible to have README.md in multiple/different languages.

I've personally have seen it in some open source repositories previously, if I recall correctly.

You can also try it out on your own README.md to test whether multiple languages are supported.

Hope this answers your question 😊

**Reply by Gabri432** [reply]
Thank you! I will try immediately.

Thank you for your kindness :)

**Reply by melvincwng** [reply]
To add on my previous comment, in those repositories I saw, the structure usually goes like this:

1. A README.md file (English)
2. Other README.md files in other languages - e.g. README-JP.md (Japanese) or README-KR.md (Korean)

Hope this helps

**Reply by melvincwng** [reply]
If this answer has helped you, please kindly 'Mark as answer' so that others know this issue has been addressed.

Take care, and have a nice day ahead  😊

**Reply by Gabri432** [reply]
Yes, I have found this repository, `https://github.com/jonatasemidio/multilanguage-readme-pattern`, that does exactly as you said.

**Reply by melvincwng** [reply]
@Gabri432 Glad to have helped Grabriele :)

**Comment by fabio-info-bsi** [comment]
> Yes, I have found this repository, `https://github.com/jonatasemidio/multilanguage-readme-pattern`, that does exactly as you said.

Excelent !

**Comment by weskerty** [comment]
I don't know how they made it work... The Spanish variant doesn't work.

**Reply by andersoncampos** [reply]
You need to create an readme.md file for each language, name it as README.es.md for spanish.. README.pt-br.md for brazilian portuguese.. etc. After this you will access it against a url link using an github badge for each language created.

**Reply by weskerty** [reply]
> Necesitas crear un archivo readme.md para cada idioma, nombrarlo como README.es.md para español.. README.pt-br.md para portugués brasileño.. etc. Después de esto, accederá a él contra un enlace url utilizando una insignia github para cada idioma creado.

Thanks, I thought Github did this automatically. I ended up using a page that does it.
https://weskerty.github.io/LinuxOneClick/
